tests/testthat/test-CDISCTealData.R

```diff
@@ -1,1205 +0,0 @@
-adsl_raw <- as.data.frame(as.list(setNames(nm = get_cdisc_keys("ADSL"))))
```

Comment:
Are there any tests here which should have been TealData tests but were actually CDISCTealData ? If so we should move them rather than delete them